tests/unit/asio_service_test.cxx

```diff
@@ -1245,11 +1245,11 @@ int leadership_transfer_test() {
     CHK_EQ(1, s3->raftServer->get_leader());
 
     // Set the priority of S2 to 10.
-    s1->raftServer->set_priority(2, 10);
+    CHK_TRUE( s1->raftServer->set_priority(2, 10) == raft_server::PrioritySetResult::SET );
```

Comment:
It exceeds 90 chars. Please replace these with
```c++
CHK_EQ( raft_server::PrioritySetResult::SET, s1.raftServer->set_priority(2, 100) );
```
like you did in the other file.